### PR TITLE
[Fix-497]-head collisions comparison

### LINKED ISSUE #497: [Bug] head_on_collisions_comparison
**Describe the bug**
There are a few bugs:
1. The 25% in the large pie is hidden (see attached screenshot)
2. The size of the pie angles doesn't match the percentage it represents (for example the red 25% takes more than a quarter in the pie itself)
3. Instead on "בכבישים עירוניים" should be "בכבישים בין עירוניים"

**To Reproduce**
See screenshot below and and link: https://anyway-infographics.web.app/newsflash/58451

**Screenshots**
Screen Shot 2020-12-05 at 17 57 22

### PR DESCRIPTION
**Pull request by almogbhl**
Resolves #497

### CODE REVIEW

File: src/components/molecules/widgets/HeadOnCollisionsComparisonWidget.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const useStyles = makeStyles(() => ({
     height: SECONDARY_CONTENT_HEIGHT,
     width: '80%',
     position: 'relative',
-    bottom: '25%',
+    bottom: '10%',
```
Comment (almogbhl) [inline src/components/molecules/widgets/HeadOnCollisionsComparisonWidget.tsx:53]: @yuvalbl -  The 25% in the large pie is hidden (see attached screenshot)

File: src/components/molecules/PieChartView.tsx [hunk 2]
```diff
@@ -125,7 +126,6 @@ export const PieChartView: FC<IProps> = ({
           nameKey={xLabel}
           innerRadius={innerRadius}
           outerRadius={outerRadius}
-          minAngle={15}
```
Comment (almogbhl) [inline src/components/molecules/PieChartView.tsx:128]: @yuvalbl - The size of the pie angles doesn't match the percentage it represents (for example the red 25% takes more than a quarter in the pie itself)

File: src/components/molecules/PieChartView.tsx [hunk 3]
```diff
@@ -128,6 +128,8 @@ export const PieChartView: FC<IProps> = ({
           outerRadius={outerRadius}
           label={usePercent ? renderLabelPercent : renderLabelCount}
           labelLine={false}
+          startAngle={-270}
```
Comment (almogbhl) [inline src/components/molecules/PieChartView.tsx:131]: start pie from 12 oclock instead of 15

File: .storybook/main.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```
Comment (yuvalbl) [inline .storybook/main.js:2]: change to TS

File: .storybook/preview.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import React, { Suspense } from 'react';
```
Comment (yuvalbl) [inline .storybook/preview.js:1]: change to TS

File: src/components/atoms/Button.stories.tsx [hunk 6]
```diff
@@ -1,9 +1,10 @@
-import React from 'react';
-import { action } from '@storybook/addon-actions';
-import Button from './Button';
+export {};
+// import React from 'react';
+// import { action } from '@storybook/addon-actions';
```
Comment (yuvalbl) [inline src/components/atoms/Button.stories.tsx:3]: remove comments

File: src/components/molecules/PieChartView.tsx [hunk 7]
```diff
@@ -125,9 +126,10 @@ export const PieChartView: FC<IProps> = ({
           nameKey={xLabel}
           innerRadius={innerRadius}
           outerRadius={outerRadius}
-          minAngle={15}
           label={usePercent ? renderLabelPercent : renderLabelCount}
           labelLine={false}
+          startAngle={-270}
+          endAngle={360}
```
Comment (yuvalbl) [inline src/components/molecules/PieChartView.tsx:132]: can be removed?

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**